### PR TITLE
Update containerservice scenarios package-2022-06

### PR DESCRIPTION
**Pull request by Alancere**
fixes: https://github.com/Azure/azure-sdk-for-go/issues/19315

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml [hunk 1]
```diff
@@ -9,41 +9,58 @@ variables:
     type: string
   azureClientSecret:
     type: string
+  
+prepareSteps:
+  - step: ManagedClusters_CreateOrUpdate
+    exampleFile: ..\examples\ManagedClustersCreate_Update.json
```
Comment (leni-msft) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml:15]: use `/`

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml [hunk 2]
```diff
@@ -9,41 +9,58 @@ variables:
...
+    requestUpdate:
+      - remove: /parameters/identity
+      - remove: /parameters/sku
+      - remove: /parameters/tags
+      - replace: /parameters/properties
```
Comment (leni-msft) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml:20]: use operation based step if you need rewrite all the requests

Comment (leni-msft) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml:20]: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/api-scenario/references/ApiScenarioDefinition.md#operation-based-step

**Review by leniatgh — CHANGES_REQUESTED** [review]

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml (hunk 1 above)
Comment (Alancere) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml]: replaced /

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml (hunk 2 above)
Comment (Alancere) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/scenarios/managedClusters.yaml]: replaced operationId

**Review by leniatgh — APPROVED** [review]

**Pull request merged**